### DISCUSSION TITLE
Support Org (and Markdown) as first-class code files

### DISCUSSION THREAD

**Discussion by mankoff** [community/community#3733] [Other Feature Feedback, Questions, & Ideas]
Here is the advice how to create links to lines in a file:
https://github.com/github/docs/blob/main/content/github/managing-your-work-on-github/creating-a-permanent-link-to-a-code-snippet.md

It does not work for Markdown or Org syntax files. Org files should render by default (THANK YOU for supporting Org format), but are also first-class code files because of Org Babel and literate programming. See for example https://github.com/nakkaya/ferret .

Please allow a method to treat Org files as simple source code, and link to lines.

**Comment by jsjoeio** [comment]
Came here to ask for the same thing! .org files render as HTML in the browser, but not in the app
